### DISCUSSION TITLE
Sudo so-test fail

### DISCUSSION THREAD

**Discussion by 418Intern** [Security-Onion-Solutions/securityonion#3090] [Unsupported Versions]
I'm having issues after installing getting any of my test be successful. Below in the link so the command sudo so-test, but I also fail the pcap test and I'm not sure what's going on. I'm using a vps (WIndows server 2016) with vmware on top of it. Any help would be appreciated. 

![Error](https://user-images.githubusercontent.com/78233142/108892330-b9c74a80-75dd-11eb-8362-677861d846f9.png)

**Comment by cm-ops** [comment]
This is being addressed in the next update (2.3.30) see https://github.com/Security-Onion-Solutions/securityonion/issues/2463.
